src/ports/postgres/modules/utilities/minibatch_preprocessing.py_in

```diff
@@ -487,10 +487,16 @@ class MiniBatchDocumentation:
         ----------------------------------------------------------------
                             SUMMARY
         ----------------------------------------------------------------
-        MiniBatch Preprocessor is a utility function to pre process the input
-        data for use with models that support mini-batching as an optimization
+        The mini-batch preprocessor is a utility that prepares input data for
+        use by models that support mini-batch as an optimization option. (This
+        is currently only the case for Neural Networks.) It is effectively a
+        packing operation that builds arrays of dependent and independent
```

Comment:
should we instead say `build matrix of independent variable(s) and arrays of dependent variable` ?